components/Text/Text.module.scss

```
@@ -1,54 +1,52 @@
-// Font Size
-.textTitle,
-.textH1,
-.textH2,
-.textH3,
-.textH4 {
-  font-weight: map-get($font-weights, bold);
-}
-
-.bodyLargeRegular,
-.bodyLargeSimeBold {
-  font-size: $font-body-large;
-}
-
-.bodySmallRegular,
-.bodySmallSemiBold {
-  font-size: $font-body-small;
-}
+.text {
+  &__title,
+  &__h1,
+  &__h2,
+  &__h3,
+  &__h4 {
+    font-weight: map-get($font-weights, bold);
+  }
```

change them to modifiers
```suggestion
  &--h1,
  &--h2,
  &--h3,
  &--h4 {
    font-weight: map-get($font-weights, bold);
  }
```